modal/gpu.py

```diff
@@ -150,7 +148,7 @@ class Any(_GPUConfig):
     """Selects any one of the GPU classes available within Modal, according to availability."""
 
     def __init__(self, *, count: int = 1):
-        super().__init__(api_pb2.GPU_TYPE_ANY, count, "ANY")
+        super().__init__(count, "ANY")
```

Comment:
Still think we should probably get rid of this, or think about a more correct way of spelling it.